### DISCUSSION TITLE
Connect to a site directly via IP?

### DISCUSSION THREAD

**Discussion by jaredcat** [EasyEngine/easyengine#1544] [Support Requests]
I have a single site hosted using EE and using ClouldFlare.  
I have a service that connects to the site but times out because of CloudFlare's low connection time.
I would like to be able to configure this service to bypass Cloudflare by using the IP address of my server instead of the domain name. 
I understand this could only be set up for 1 site at a time which is fine for my situation.
When trying to visit the site directly i get an nginx 404 error. 

What would i need to change so the site resolves on the IP of the server?

**Comment by robbie-berns** [comment]
Hi. If I have understood you correctly, the easiest way is to add an entry in your /etc/hosts file on your PC. This will bypass the DNS lookup.

**Reply by jaredcat** [reply]
I cannot edit the hosts file on another service i don't own.  
I want to add the raw ip as a valid server to the main nginx service

**Reply by robbie-berns** [reply]
OK, and this is from a distant memory: use URL https://<ip-address>/<site name> will get you to the base server. 
Alternatively 

- you can set up a specific port on the server (https://serverfault.com/questions/351559/nginx-without-server-name-and-using-only-static-ip-address) or
- look at the bottom of https://www.linode.com/community/questions/9192/how-can-i-access-my-nginx-website-via-its-ip-address

**Reply by jaredcat** [reply]
So im pretty sure i need to edit `/opt/easyengine/services/nginx-proxy/conf.d` but when i do and reload it's overwritten. So im running into an issue with lack of understanding of how ee is managing the files for the nginx proxy

**Reply by robbie-berns** [reply]
I believe that you need to do mods in /var/lib/docker/volumes/<sitename>_config_nginx/_data/custom/user.conf.

_ee site reload <sitename>_ will rewrite the default structure into .../_data/conf.d/main.cnf

**Comment by jaredcat** [comment]
Wasn't able to get this working but adding `/wp-json/*` to a cloudflare rule that bypassed security and cache settings helped.
I then edited the php settings in increase script execution time limit as that was probably the true root cause stopping the remote connection
